<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddTeacherDialog</class>
 <widget class="QDialog" name="AddTeacherDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>435</width>
    <height>533</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(222, 221, 218);
color: rgb(0, 0, 0);</string>
  </property>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>50</y>
     <width>561</width>
     <height>31</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Inter</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Add Teacher</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>171</width>
     <height>54</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <family>Inter</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>name</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="name_field"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>20</y>
     <width>241</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="add_teacher_button">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(52, 212, 129);
color:white;
border:none;
border-radius: 8px;
font-weight:bold;
font-size: 15px;</string>
      </property>
      <property name="text">
       <string>Add Teacher</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(88, 88, 88);
color:white;
border:none;
border-radius: 8px;
font-weight:bold;
font-size: 15px;</string>
      </property>
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>80</y>
     <width>86</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <strikeout>false</strikeout>
    </font>
   </property>
   <item>
    <property name="text">
     <string>M</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>F</string>
    </property>
   </item>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>120</y>
     <width>171</width>
     <height>54</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="font">
       <font>
        <family>Inter</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>last name</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="last_name_field"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>361</width>
     <height>54</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <family>Inter</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>email</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="email_field"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>260</y>
     <width>361</width>
     <height>54</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QLabel" name="label_5">
      <property name="font">
       <font>
        <family>Inter</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>phone</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="phone_field"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>340</y>
     <width>361</width>
     <height>54</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <widget class="QLabel" name="label_6">
      <property name="font">
       <font>
        <family>Inter</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>address *</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="address_field"/>
    </item>
   </layout>
  </widget>
  <widget class="QComboBox" name="subject_dropdown">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>86</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="subjects_widget" native="true">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>390</y>
     <width>251</width>
     <height>131</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(153, 193, 241);</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
